bin/setup

```diff
@@ -3,9 +3,10 @@ require "fileutils"
 
 # path to your application root.
 APP_ROOT = File.expand_path("..", __dir__)
+APP_NAME = "forms-api"
```

Comment:
Similar to runner, do we need this or is it best to keep it as the update task will add it anyway?

Reply:
Keeping